### PR TITLE
e2e RI-3637 databases import

### PR DESCRIPTION
**Pull request by vlad-dargel**

### CODE REVIEW

File: tests/e2e/pageObjects/my-redis-databases-page.ts [hunk 1]
```diff
@@ -30,6 +30,11 @@ export class MyRedisDatabasePage {
     sortByHostAndPort = Selector('span').withAttribute('title', 'Host:Port');
     sortByConnectionType = Selector('span').withAttribute('title', 'Connection Type');
     sortByLastConnection = Selector('span').withAttribute('title', 'Last connection');
+    importDatabasesBtn = Selector('[data-testid=import-dbs-btn]');
+    submitImportBtn = Selector('[data-testid=submit-btn]');
+    closeDialogBtn = Selector('[aria-label="Closes this modal window"]');
```
[the remark this answers is not in the document]
Comment (vlad-dargel) [reply tests/e2e/pageObjects/my-redis-databases-page.ts:35]: no, it's Elastic components which can be described in code

File: tests/e2e/common-actions/databases-actions.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        for (const db of databases) {
+            const databaseName = myRedisDatabasePage.dbNameList.withText(db);
+            await t.expect(databaseName.exists).ok(`"${db}" database doesn't exist`);
+        }
+    }
+
+    /**
+     * Import database using file
+     * @param pathToFile The path to file for import
+     * @param databaseType The database type
```
Comment (ofersteinberg) [inline tests/e2e/common-actions/databases-actions.ts:21]: What is that? why not creating a type if there are several options?

File: tests/e2e/test-data/ardm-valid.ano [hunk 3]
```diff
@@ -0,0 +1 @@
+W3siaG9zdCI6ImxvY2FsaG9zdCIsInBvcnQiOiI2Mzc5IiwiYXV0aCI6InBhc3MiLCJ1c2VybmFtZSI6InVzZXJuYW1lVGVzdCIsIm5hbWUiOiJhcmRtV2l0aFBhc3NBbmRVc2VybmFtZSIsInNlcGFyYXRvciI6IjoiLCJjbHVzdGVyIjpmYWxzZSwia2V5IjoiMTY1MDM3MzUyNTY1MV9naHFwciIsIm9yZGVyIjowfSx7Imhvc3QiOiJhcmRtTm9OYW1lIiwicG9ydCI6IjEyMDAxIiwiYXV0aCI6IiIsInVzZXJuYW1lIjoiIiwic2VwYXJhdG9yIjoiOiIsImNsdXN0ZXIiOmZhbHNlLCJrZXkiOiIxNjUwODk3NjIxNzU0X2I1bjB2Iiwib3JkZXIiOjF9XQ==
```
Comment (ofersteinberg) [inline tests/e2e/test-data/ardm-valid.ano:1]: ?

File: tests/e2e/tests/critical-path/database/import-databases.e2e.ts [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+        // Delete databases
+        deleteStandaloneDatabasesByNamesApi(databases);
+    })('Connection import from JSON', async t => {
+        const tooltipText = 'Import Database Connections';
+        const partialImportedMsg = 'Successfully added 2 of 6 database connections';
+        const defaultText = 'Select or drag and drop a file';
+
+        // Verify that user can see the “Import Database Connections” tooltip
+        await t.hover(myRedisDatabasePage.importDatabasesBtn);
+        await t.hover(myRedisDatabasePage.importDatabasesBtn);
```
Comment (ofersteinberg) [inline tests/e2e/tests/critical-path/database/import-databases.e2e.ts:62]: Why twice?

File: tests/e2e/tests/critical-path/database/import-databases.e2e.ts [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+        await t.expect(myRedisDatabasePage.importDbDialog.textContent).contains(rdmData.fileName, 'Filename not displayed in import input');
+        // Click on remove button
+        await t.click(myRedisDatabasePage.removeImportedFileBtn);
+        await t.expect(myRedisDatabasePage.importDbDialog.textContent).contains(defaultText, 'File not removed from import input');
+
+        // Verify that user can import database with mandatory fields
+        await t.click(myRedisDatabasePage.closeDialogBtn);
+        await databasesActions.importDatabase(rdmData.path);
+        // Verify that success message is displayed
+        await t.expect(myRedisDatabasePage.successImportMessage.textContent).contains(partialImportedMsg, 'Successfully added databases number not correct');
```
Comment (ofersteinberg) [inline tests/e2e/tests/critical-path/database/import-databases.e2e.ts:90]: The error message doesn't clear

File: tests/e2e/common-actions/databases-actions.ts (hunk 2 above)
Comment (vlad-dargel) [reply tests/e2e/common-actions/databases-actions.ts:21]: thanks, created ImportDatabaseParameters type

File: tests/e2e/test-data/ardm-valid.ano (hunk 3 above)
Comment (vlad-dargel) [reply tests/e2e/test-data/ardm-valid.ano:1]: this is .ano file for upload (it is in base64 format)

File: tests/e2e/tests/critical-path/database/import-databases.e2e.ts (hunk 5 above)
Comment (vlad-dargel) [reply tests/e2e/tests/critical-path/database/import-databases.e2e.ts:90]: fixed

File: tests/e2e/tests/critical-path/database/import-databases.e2e.ts (hunk 4 above)
Comment (vlad-dargel) [reply tests/e2e/tests/critical-path/database/import-databases.e2e.ts:62]: first hover method sometimes wasn't working properly, fixed

**Review by ofersteinberg — APPROVED** [review]

**Pull request merged**